packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -100,7 +100,7 @@ class Authenticator extends StatefulWidget {
     Key? key,
     SignInForm? signInForm,
     SignUpForm? signUpForm,
-    ConfirmSignInMFAForm? confirmSignInMFAForm,
```

Comment:
I am not sure if we need to support a customizable MFA screen, but it wasn't really supported since `ConfirmSignInMFAForm` only has a default constructor. I think this was added when `ConfirmSignInMFAForm` and `ConfirmSignInNewPasswordForm` were one class, but now they are separate.